opentelemetry/instrumentation/app/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM golang:1.21
+WORKDIR /usr/src/app
+COPY go.mod go.sum ./
+RUN go mod download && go mod verify
+COPY *.go ./
+RUN go build -v -o /usr/local/bin/app .
+CMD sh -c "app | tee /var/log/app.log"
```

Comment:
I think this will lose stderr logs, not sure if the example makes any

Reply:
fixed by adding `2>&1`